### PR TITLE
Stock Span Problem--> Closes : [#1684]

### PR DESCRIPTION
**Pull request by Prerna-07**
Stock Span Problem
Explanation : Stock span is defined as the number of days passed between the current day and the first day before today when price was higher than today.

closes : #1684

### CODE REVIEW

**Comment by Prerna-07** [comment]
@nakul-19 @raksha009 @plazzy99 @tarun26091999 @rish-singhal @yasharth291 @Rohan27s @rudrakshi99 pls check

File: Code/Java/Balanced Bracket Problem.java [hunk 1]
```diff
@@ -1,82 +0,0 @@
-/* Balanced Bracket Problem
-Explanation :- To check if the expression is balanced i.e. closing brackets and opening brackets match up well.
-It will be an iterative solution in which loop will iterate over each character of expression, if opening brackets encountered then simply push in stack, if closing brackets
```
Comment (Rohan27s) [inline Code/Java/Balanced Bracket Problem.java:3]: Why have you deleted someone else's code? Revert it back

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by Prerna-07** [comment]
@Rohan27s sorry sir it it my code i didn't deleted it.

**Comment by Prerna-07** [comment]
@Rohan27s added the file in this branch sir pls check.

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**